modules/internal/fixInternalDocs.sh

```diff
@@ -131,17 +131,6 @@ removeUsage $file
 ## End ChapelIO ##
 
 
-## ChapelIteratorSupport ##
-
-file="./ChapelIteratorSupport.rst"
-removePrefixFunctions $file
-fixTitle "Vectorizing Iterator" $file
-removeUsage $file
```

Comment:
The current documentation renders as "to get this `use VectorizingIterator`, but note that it's auto-used so you don't need to `use` it", which seems a little odd

Reply:
e.g.:
<img width="748" alt="Screen Shot 2020-12-14 at 10 25 28 AM" src="https://user-images.githubusercontent.com/1588337/102100143-198fc100-3df7-11eb-9154-764807a846c1.png">

Reply:
Right, we have other modules that look the same way, e.g. https://chapel-lang.org/docs/master/modules/standard/Math.html

I think it's reasonable to fix that but I wasn't viewing it as a requirement for this PR specifically.

Reply:
Gotcha -- could you create an issue to capture that?

Reply:
PR #16861 will do it.